docs/source/demos/lead_scoring.ipynb

```diff
@@ -146,6 +146,7 @@
     "                      problem_type='binary',\n",
     "                      objective=lead_scoring_objective,\n",
     "                      additional_objectives=['auc'],\n",
+    "                      allowed_model_families=[\"catboost\", \"random_forest\", \"linear_model\"],\n",
```

Comment:
Seems like the only way to not fail the lead_score >= auc score assertion is to train catboost on the entire data. I think this is the cleanest way to do that while also cutting out some pipelines to not spend about a minute on this search.

Reply:
good idea!